.cloud-build/execute_changed_notebooks_cli.py

```diff
@@ -129,26 +136,33 @@ def str2bool(v):
     base_branch=args.base_branch,
 )
 
+
+results_bucket = f"gs://{args.artifacts_bucket}"
```

Comment:
results_bucket = f"gs://{args.artifacts_bucket}/build_results"

# Download all build logs 
# Combine into one array of NotebookExecutionResult